docker/uwsgi-django-shop/requirements.txt

```diff
@@ -29,6 +29,7 @@ djangocms-bootstrap3==0.1.1
 djangocms-cascade==0.8.2
 djangocms-text-ckeditor==2.8.1
 djangorestframework==3.1.3
+https://github.com/django-haystack/django-haystack/archive/master.zip
```

Comment:
This is not good. We should pin to an explicit version or commit. Otherwise, we do not know which version of Haystack we will end up with.